### ISSUE TITLE
plugin itself is not accessible

### ISSUE THREAD

**Issue by Psychopomp777** [pojome/one-click-accessibility#79]
I find it very strange that an accessibility plugin itself is not accessible. The default font size in the toggle is way too small and can't be changed in the plugin settings. Also the width of the toogle box cannot be changed. Got it working now with custom code, but this solution is not clean (it took a lot of !importants).

**Comment by Psychopomp777** [comment]
And: although I disabled some tools (e.g. High Contrast), they are still displayed on the page. That seems to be a bug!

**Comment by sasgrw** [comment]
@Psychopomp777, there aren't any WCAG requirements regarding font size so technically it's not an accessibility failure.  However, if I use the "increase font" option within the tool, or use the browser's builtin font change (ctrl++ or cmd++), the font of the tool increases.

**Comment by arielk** [comment]
You issue was fixed: https://github.com/pojome/one-click-accessibility/issues/39

Thanks for your feedback!
